pkg/trace/agent/agent_test.go

```diff
@@ -328,7 +327,7 @@ func TestProcess(t *testing.T) {
 		go agnt.Process(&api.Payload{
 			Traces: traces,
 			Source: agnt.Receiver.Stats.GetTagStats(info.Tags{}),
-		}, stats.NewSublayerCalculator())
+		})
```

Comment:
I think it would be better to drop sublayers in another PR, on which you would base stats computation.
It makes it harder to read and it's actually worth a release note on its own

Reply:
Makes sense.